paimon-core/src/main/java/org/apache/paimon/utils/TagManager.java

```diff
@@ -228,6 +249,11 @@ public SortedMap<Snapshot, List<String>> tags() {
         return tags(tagName -> true);
     }
 
+    /** Get all tagged snapshots with names sorted by snapshot id. */
+    public SortedMap<Snapshot, List<String>> tagsWithBranch(String branchName) {
```

Comment:
This is not used.

Reply:
> This is not used.

I deleted it, which was previously used by the merge branch